tools/metadata_generation/swapi_metadata_generator.py

```diff
@@ -0,0 +1,97 @@
+"""SWAPI metadata generator."""
+
+import json
+
+import pandas as pd
+
+
+def get_global_attributes(sheet):
+    """Get the global attributes from a metadata spreadsheet.
+
+    Parameters
+    ----------
+    sheet : pandas.DataFrame
+        A sheet from an Excel file represented as a pandas DataFrame.
+
+    Returns
+    -------
+    dict
+        A dictionary of the global attributes, where the keys are the attribute names
+        and the values are the attribute descriptions.
+    """
+    sheet = sheet.set_index("NAME")
+    # subset by global_att field type, then only keep the CATDESC column
+    sheet = sheet[sheet["FIELD_TYPE"] == "global_att"][["CATDESC"]]
+    return sheet.to_dict()["CATDESC"]
+
+
+def get_file_attributes(sheet, global_attrs=None):
```

Comment:
Should this be called `get_sheet_attributes`?

Reply:
Good idea to update this. I opted for `get_dataset_attributes` as I think each "sheet" we are interested in here is a dataset. But the `global_attrs` is also a sheet, so maybe that is a better distinguisher? Let me know if you have a strong preference for "sheet" here and we can use that too as it is much better than file.

Reply:
I agree... `get_dataset_attributes` is better yet.